components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/RefreshGrantHandler.java

```diff
@@ -142,7 +144,12 @@ public OAuth2AccessTokenRespDTO issue(OAuthTokenReqMessageContext tokReqMsgCtx)
 
         tokReqMsgCtx.setValidityPeriod(validationBean.getAccessTokenValidityInMillis());
 
-        executePreIssueAccessTokenActions(validationBean, tokReqMsgCtx);
+        ActionExecutionStatus<?> executionStatus = executePreIssueAccessTokenActions(validationBean, tokReqMsgCtx);
+        // TODO: Remove the null check for the executionStatus once the unit tests are onboarded.
```

Comment:
This todo is now redundant right based on the conversation at https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2609/files#r1828711448

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2609/commits/5ff70df701d60720abd41de1bada6a7c85d4d659